File: PROGRAM_CHECK_SYSTEM/src/main/java/ru/doctorixx/core/structures/ProcessEndStatus.java
package ru.doctorixx.core.structures;

public enum ProcessEndStatus {
    SUCCESS("OK"),
    RUNTIME_ERROR("RE"),
    COMPILE_ERROR("CE"),
    TIME_LIMIT("TL"),
    WRONG_ANSWER("WA"),
    NOT_EXECUTED("NE"),
    ;

    private final String msg;

    ProcessEndStatus(String msg){

        this.msg = msg;
    }

    @Override
    public String toString() {
        return msg;
    }
}

File: BACKEND_V2/src/main/kotlin/ru/codebattles/backend/web/entity/checker/ProcessEndStatus.kt
package ru.codebattles.backend.web.entity.checker

import io.swagger.v3.oas.annotations.media.Schema

@Schema(description = "Status of the process execution")
enum class ProcessEndStatus(private val msg: String) {
    SUCCESS("OK"),
    RUNTIME_ERROR("RE"),
    COMPILE_ERROR("CE"),
    TIME_LIMIT("TL"),
    WRONG_ANSWER("WA"),
    NOT_EXECUTED("NE"),
    ;

    override fun toString(): String {
        return msg
    }
}